src/components/SimpleList/components/Body.js

```diff
@@ -0,0 +1,65 @@
+import React, { useContext, Fragment } from 'react';
+import Paginator from '../../Paginator/index.js';
+import SimpleListItem from './SimpleItem.js';
+import { SimpleListContext } from '../index';
+import ButtonOrDiv from './ButtonOrDiv';
+
+const Body = ({
+  SimpleItemComponent = SimpleListItem,
+  PaginatorComponent = Paginator,
+}) => {
+  const {
+    name,
+    headings,
+    changePage,
+    changePageLimit,
+    pageData: { lastPage, currentPage, perPage },
+    data,
+    initialLoad,
+    childrenRenderer,
+    allowClick,
+    onItemClick,
+  } = useContext(SimpleListContext);
+  return (
+    <Fragment>
+      <div className="simple-list__body">
+        {typeof childrenRenderer === 'function'
+          ? childrenRenderer(data)
+          : data.map((v: Object, vk: number): React$Element<*> => (
+              <ButtonOrDiv
+                key={vk}
+                allowClick={allowClick}
+                index={vk}
+                clickAct={onItemClick}
+                item={v}
+                identifier={`${name}-table-${vk}`}>
+                {headings.map((col: Object, ck: number): React$Element<*> => (
+                  <SimpleItemComponent
+                    key={`${ck}`}
+                    text={v[col.name]}
+                    flex={`${(1 / headings.length) * 100}%`}
+                    align={col.align}
+                    column={col.name}
+                    itemClass="simple-list__body__row__item"
+                    customFormatter={col.customFormatter || null}
```

Comment:
hmm, this might get annoying if you have the same formatter for all of your lists (like allianz).

Should we not support a generic formatter as a prop on SimpleList instead of a property on columns?

Reply:
no, it appears that sometimes we just do custom currency formatting, because we need to aggregate the data (amounts) in group headings (so we can't employ tagged values)

Reply:
but then if we do employ a more generic formatter then, we'd have to send values in two formats sometimes...

Reply:
okidoki